actors/builtin/singletons.go

```diff
@@ -8,12 +8,12 @@ import (
 // Addresses for singleton system actors.
 var (
 	// Distinguished AccountActor that is the source of system implicit messages.
-	SystemActorAddr        = mustMakeAddress(0)
-	InitActorAddr          = mustMakeAddress(1)
-	RewardActorAddr        = mustMakeAddress(2)
-	CronActorAddr          = mustMakeAddress(3)
-	StoragePowerActorAddr  = mustMakeAddress(4)
-	StorageMarketActorAddr = mustMakeAddress(5)
+	InitActorAddr          = mustMakeAddress(0)
+	RewardActorAddr        = mustMakeAddress(1)
+	StoragePowerActorAddr  = mustMakeAddress(2)
+	StorageMarketActorAddr = mustMakeAddress(3)
+	CronActorAddr          = mustMakeAddress(4)
+	SystemActorAddr        = mustMakeAddress(5)
```

Comment:
This seems like an unnecessary change to me. The proposed ordering is somewhat arbitrary history-dependent order, the original somewhat builds up from foundations to application-level.

Reply:
I worry that users in the community already have things set up to interact with system actors. Forcing that change will be disruptive, but I won't push the point any further